screens/AllProductsScreen.js

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import { View, FlatList } from 'react-native';
+import { ScrollView } from 'react-native-gesture-handler';
+
+import Product from '../components/Product';
+import BASE from '../lib/common';
+import { Button, ScrollCategory, H3, Title, Subtitle } from '../styles/shared';
+import { styles } from '../styles/products';
+
+const productsTable = BASE('Products').select({ view: 'Grid view' });
+let fullProducts;
+const categories = [
+  // Hard-coded for now -- should find a way to extract this information dynamically?
```

Comment:
@anniero98 

Unfortunately you can't query the possible options for multi-select fields -- but if you did wanted to get your categories dynamically you would need to make an additional table and do some record linking.  

One possible solution -- make a `Category` table with only 1 column `name` that has the category names   
  'All',
  'Cut Fruit & Packaged Products',
  'Fruit',
  'Vegetables',
  'Frozen & Dried'
And in your `products` table you make two columns -- one (you could name it `CategoryLink` will will link to the `category` 
<img width="310" alt="Screen Shot 2019-11-30 at 11 27 35 PM" src="https://user-images.githubusercontent.com/4732011/69910957-0b782980-13c9-11ea-9192-11d75c238a7d.png">

The other column, which you will actually name `category` (which will replace your current category column -- you can name it something else in the mean time) will be a of this type
<img width="317" alt="Screen Shot 2019-11-30 at 11 30 34 PM" src="https://user-images.githubusercontent.com/4732011/69910985-7f1a3680-13c9-11ea-85ca-4470abf062ca.png">
So now, if you want to just query all the categories you can now just query the Category table, and this will also maintain the `category` field in Products.

Although I see that you only handle Products with one category?

Reply:
Originally, we thought of having other categories such that they could be a multi-select (such as in-season produce, etc). Currently that idea has been scrapped, so we could indeed make it its own table & link the records!

There are few enough, though, that right now I think the global constants is a good approach. I'll note it down in a list of future works~! Wanted to get this PR merged in first - we'll refactor soon